lib/src/chat/chat.dart

```diff
@@ -827,10 +827,22 @@ class _ChatState extends State<Chat> with ChatComposer, ChatCreateMixin, InviteM
 
   _getFilePath(FileType fileType, [String extension]) async {
     _navigation.pop(context);
+
     String filePath = await FilePicker.getFilePath(type: fileType, fileExtension: extension);
     if (filePath == null) {
       return;
     }
+
+    if (fileType == FileType.video) {
+      final ext = Path.extension(filePath);
+      final videoFileName = "${filePath.hashCode}$ext";
+      final videoFileDir = Path.dirname(filePath);
+      final videoFilePath = "$videoFileDir/$videoFileName";
```

Comment:
Looks all good, just one thing, let's use https://api.dart.dev/stable/2.7.2/dart-io/Platform/pathSeparator.html instead of "/" to concatenate paths..

Reply:
Great advice, didn't know that!

Reply:
@Boehrsi Changes done.